### PR TITLE
Add XImageSharing bypass

### PR DESCRIPTION
**Pull request by TechnoJo4**
<details> <summary>NOTICE</summary>
I dedicate any and all copyright interest in this software to the
public domain. I make this dedication for the benefit of the public at
large and to the detriment of my heirs and successors. I intend this
dedication to be an overt act of relinquishment in perpetuity of all
present and future rights to this software under copyright law.
</details>

Adds a bypass for websites "Powered by [XImageSharing](https://sibsoft.net/ximagesharing.html)" (e.g. imgsto.com, picbaron.com, and many more)

- [x] I made sure there are no unnecessary changes in the code*
- [ ] Tested on Chromium- Browser OS
- [x] Tested on Firefox

\* indicates required

### CODE REVIEW

**Comment by NotAProton** [comment]
Any example links?

File: src/js/injection_script.js [hunk 1]
```diff
@@ -2325,6 +2325,16 @@ ensureDomLoaded(()=>{
 	})
 	domainBypass("techynroll.com", ()=>awaitElement("a#enablebtn", a=>safelyAssign(a.href)))
 	hrefBypass(/meostream\.com\/links\//,()=> ifElement("a#link",safelyNavigate))
+	//XImageSharing
+	ifElement('input[type=submit][value="Continue to image..."]', submit => {
+	    submit.click()
```
Comment (NotAProton) [inline src/js/injection_script.js]: tabs, not spaces

```suggestion
		submit.click()
```

**Review by NotAProton — CHANGES_REQUESTED** [review]

**Review by NotAProton — APPROVED** [review]
LGTM

**Comment by NotAProton** [comment]
Thanks!

**Comment by lem6ns** [comment]
closed?

**Comment by NotAProton** [comment]
> 
> 
> closed?

Accidentally clicked close with comment, sorry about that

**Pull request merged**